modules/statics/src/coins.ts

```diff
@@ -7502,6 +7502,70 @@ export const coins = CoinMap.fromCoins([
     '0xc770eefad204b5180df6a14ee197d99d808ee52d',
     UnderlyingAsset.FOX
   ),
+  erc20(
+    '523d92b8-5fc8-4d13-8359-b69f3ac64382',
+    'cube',
+    'CUBE',
+    18,
+    '0x622dffcc4e83c64ba959530a5a5580687a57581b',
+    UnderlyingAsset.CUBE
+  ),
+  erc20(
+    'ae798fd0-30cc-47de-af7d-cae0e3db15ad',
+    'xx',
+    'XX',
+    8,
+    '0x47f33feb70ec153fc22755908a4061777d22c2da',
+    UnderlyingAsset.XX
+  ),
+  erc20(
+    'd340bbfc-1215-418d-8e57-64ffc954e2d7',
+    'litt',
+    'LITT',
+    18,
+    '0xc5b3d3231001a776123194cf1290068e8b0c783b',
+    UnderlyingAsset.LITT
+  ),
+  erc20(
+    'fdaf03ca-1177-4284-81f3-b931721e0208',
+    'media-network-token',
+    'Media Network Token',
+    6,
+    '0xdb726152680ece3c9291f1016f1d36f3995f6941',
+    UnderlyingAsset.MEDIANETWORK
+  ),
+  erc20(
+    '2b2800aa-42f3-4bf4-9f15-35f76aace2a8',
+    'blurr',
+    'BLUR',
+    6,
+    '0xb9340fb0f8f93f4857eb3012b4f85db2d632e154',
+    UnderlyingAsset.BLURR
```

Comment:
```suggestion
    UnderlyingAsset.BLUR
```

Reply:
Same thing with this one too - https://github.com/BitGo/BitGoJS/blob/master/modules/statics/src/coins.ts#L2001

We already had BLUR already, so had to add it with the extra letter to differentiate